### ISSUE TITLE
Integrate Next.js news archive code from POC to energyvision repository

### ISSUE THREAD

**Issue by nilsml** [equinor/energyvision#243]
We should include the code from the POC Kalyan has made in the envis repo.
It is important that we don't build all the pages on every commit or push to GitHub though.

**Comment by kalyanvaranasi** [comment]
@nilsml I will start setting up the energy vision project and move the code used in POC

**Comment by nilsml** [comment]
Sounds good @kalyanvaranasi! Perhaps create a PR for other devs on the team to review before we push it to the main branch?!

**Comment by kalyanvaranasi** [comment]
@nilsml Sure Magne I will create a PR 👍

**Comment by kalyanvaranasi** [comment]
@nilsml As @vnys is currently working on loading style resources automatically I am moving this to hold unit its resolved

**Comment by nilsml** [comment]
Is this still on hold @vnys and @kalyanvaranasi?

**Comment by kalyanvaranasi** [comment]
@nilsml Yes we have to keep it on hold till we fix the styling and once that is done we can test and close it

@vnys FYI

**Comment by kalyanvaranasi** [comment]
@nilsml Radix build is success and archive pages are loading with the styles. But i have observed couple of issues for which i will create new issues.

@vnys FYI
